docs/pages/eas/environment-variables/usage.mdx

```diff
@@ -0,0 +1,176 @@
+
+> **warning** **Environment variables are per deployment, and deployments are immutable**. This means that after changing an environment variable, you will need to re-export your project, and re-deploy in order for them to be updated.
+
+### For local development
+
+For local development, both client- and server-side environment variables are loaded from [local **.env** files](/guides/environment-variables/), which should be gitignored. If you are using EAS environment variables, use [`eas env:pull`](/eas/environment-variables/manage/#pull-variables-for-local-development) to retrieve the environment variables for `development`, `preview`, or `production`.
+
+## Using environment variables for other commands
+
+One way to supply non-secret EAS environment variables to other EAS commands is to use the `eas env:exec` command.
+
+<Terminal cmd={["$ eas env:exec --environment production 'echo $APP_VARIANT'"]} />
+
+For example, it can be useful when uploading source maps to Sentry using a [`SENTRY_AUTH_TOKEN`](/guides/using-sentry) variable after an update bundle is created.
+
+<Terminal
+  cmd={["$ eas env:exec --environment production 'npx sentry-expo-upload-sourcemaps dist'"]}
+/>
+
+## EAS Build and Workflows job runs
+
+### Setting the environment for your builds
+
+To set the environment for your EAS Build jobs, you can use the `environment` option in **eas.json** and set it to one of the available environments: `development`, `preview` or `production`.
+
+```json eas.json
+{
+  "build": {
+    "development": {
+      "environment": "development"
+    },
+    "preview": {
+      "environment": "preview"
+    },
+    "production": {
+      "environment": "production"
+    },
+    "my-profile": {
+      "environment": "production"
+  }
+}
+```
+
+If you don't set the `environment` option, we will set the environment automatically based on your build's configuration:
+
+- `development`: for development client build
+- `preview`: for other configurations
+- `production`: for app store build
+
+<Collapsible summary="Built-in environment variables">
+
+The following environment variables are additional system environment variables exposed to each job and can be used within any build step. They are not a part of any project environment and are not available when evaluating **app.config.js** locally:
+
+- `CI=1`: Indicates this is a CI environment
+- `EAS_BUILD=true`: Indicates this is an EAS Build environment
+- `EAS_BUILD_PLATFORM`: Either `android` or `ios`
+- `EAS_BUILD_RUNNER`: Either `eas-build` for EAS Build cloud builds or `local-build-plugin` for [local builds](/build-reference/local-builds/)
+- `EAS_BUILD_ID`: The build ID, for example, `f51831f0-ea30-406a-8c5f-f8e1cc57d39c`
+- `EAS_BUILD_PROFILE`: The name of the build profile from **eas.json**, for example, `production`
```

Comment:
```suggestion
- `EAS_BUILD_PROJECT_ID`: the EAS app (project) ID, for example, `bd2f7e21-1ee7-47f2-8357-d7c4b50622fb`
- `EAS_BUILD_PROFILE`: The name of the build profile from **eas.json**, for example, `production`
```

Because of https://github.com/expo/expo/pull/41772/files